Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -2,178 +2,251 @@ import Foundation
 import Sentry
 import CocoaLumberjack
 
+struct CrashLoggingInternals {
```

Comment:
This class isn't fully untangled from all the `static` business, so we keep an internal-only static reference around until we can get rid of it.

Reply:
I like this idea a lot – this is done in 389e8c17c6adb2af693c522bdb3352bba0ef2278